test/remote/gateways/remote_s5_test.rb

```diff
@@ -0,0 +1,136 @@
+require 'test_helper'
+
+class RemoteS5Test < Test::Unit::TestCase
+  def setup
+    @gateway = S5Gateway.new(fixtures(:s5))
+
+    @amount = 100
+    @credit_card = credit_card('4000100011112224')
+    @declined_card = credit_card('4000300011112220')
+
+    @options = {
+      order_id: '1',
+      billing_address: address,
+      description: 'Store Purchase'
+    }
+  end
+
+  def test_dump_transcript
```

Comment:
Yeah, definitely doesn't seem clean to have a method with just comments in it. I wasn't sure if it was supposed to be kept around in case someone wanted to be able to quickly generate a transcript again, but, if there isn't a formal convention around it, I'd say whack as well.